py3status/py3.py

```diff
@@ -42,6 +44,25 @@ def __init__(self, module=None, i3s_config=None):
                 config = self._module.i3status_thread.config['general']
                 self._i3s_config = config
 
+    def __getattr__(self, name):
+        """
+        Py3 can provide COLOR constants
+        eg COLOR_GOOD, COLOR_BAD, COLOR_DEGRADED
+        but also any constant COLOR_XXX we find this color in the config
+        if it exists
+        """
+        if not name.startswith('COLOR_'):
+            raise AttributeError
+        name = name.lower()
+        if name not in self._colors:
+            if self._module:
+                color_fn = self._module._py3_wrapper.get_config_attribute
+                color = color_fn(self._module.module_full_name, name, 'color')
+            else:
+                color = self._i3s_config.get(name)
```

Comment:
yes but that is only if we do stand alone testing so I'm ok with that as they could add to test config if they wanted